dataline-server/src/main/java/io/dataline/server/handlers/DestinationImplementationsHandler.java

```diff
@@ -154,12 +153,6 @@ private void validateDestinationImplementation(
           String.format(
               "The provided configuration does not fulfill the specification. Errors: %s",
               e.getMessage()));
-    } catch (ConfigNotFoundException e) {
```

Comment:
@cgardens this exception is never thrown, I could remove this catch. Is that expected?